Library/Formula/bacula-fd.rb

```diff
@@ -13,6 +13,29 @@ class BaculaFd < Formula
   depends_on "readline"
   depends_on "openssl"
 
+  def plist; <<-EOS.undent
+    <?xml version="0.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>UserName</key>
+        <string>root</string>
```

Comment:
Yes it can, the question is how much sense it would make to run a backup agent without root access. I added a comment around the user key/value, so one is able to enable it.